index.html

```diff
@@ -1,14 +1,89 @@
 <!DOCTYPE html>
 <html lang="kr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>🏎️ 자동차 경주 게임</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <!-- font - Roboto -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Reset CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/reset-css@5.0.1/reset.min.css"
+    />
+    <link rel="stylesheet" href="./src/css/index.css" />
+  </head>
 
-<head>
-  <meta charset="UTF-8" />
-  <title>🏎️ 자동차 경주 게임</title>
-</head>
-
-<body>
-  <div id="app">
-  </div>
-</body>
+  <body>
+    <div id="app">
+      <header>
+        <h4 id="title">🏎 자동차 경주 게임🏁</h1>
```

Comment:
```suggestion
        <h1 id="title">🏎 자동차 경주 게임🏁</h1>
```
h4 태그와 h1 태그가 같이 있네요~ 수정하시는게 좋겠어요.

Reply:
네  h1으로 수정하겠습니다. 감사합니다~!